fmt/testdata/EventDefinition/fmt.sol

```diff
@@ -21,24 +21,10 @@ contract Events {
         anonymous;
 
     event Event1(
-        uint256,
-        uint256,
-        uint256,
-        uint256,
-        uint256,
-        uint256,
-        uint256,
-        uint256
+        uint256, uint256, uint256, uint256, uint256, uint256, uint256, uint256
     );
     event Event1(
-        uint256,
-        uint256,
-        uint256,
-        uint256,
-        uint256,
-        uint256,
-        uint256,
-        uint256
+        uint256, uint256, uint256, uint256, uint256, uint256, uint256, uint256
```

Comment:
the wrapped `anonymous` looks a bit lost imo, should `anonymous` events wrap earlier?

Reply:
agree that it looks a bit weird. could you clarify what's you suggestion pls?

Reply:
okay, i will revert to the previous behavior  👍